docs/commands_v2.rst

```diff
@@ -0,0 +1,313 @@
+  command. Most commonly this is a weekly load name, but it can also indicate
+  a non-load command event for which further details are provided in the command
+  parameters.
+- Information about each distinct observation is embedded into the command
+  archive as ``LOAD_EVENT`` pseudo-commands. The
+  :func:`~kadi.commands.observations.get_observations` provides a fast and
+  convenient way to find observations, both past and planned. See the
+  :ref:`getting-observations` section for more details.
+- Information about each ACA star catalog is stored in the command
+  archive. The :func:`~kadi.commands.observations.get_observations` provides a
+  convenient way to find ACA star catalogs, both past and planned. See the
+  :ref:`getting-star-catalogs` section for more details.
+- There are configuration options which can be set programmatically or in a fixed
+  configuration file to control behavior of the package. See the
+  `Configuration options`_ section for more details.
+
+Scenarios
+---------
+
+A scenario is an specific version of events that you like to evaluate. The
+default scenario is the `Chandra Command Events`_ sheet.
+
+Providing for alternate scenarios is a key feature of the commands archive v2.
+An example is checking for thermal propagation for assuming an ACIS CTI using
+either 3-chips or 4-chips, or no CTI at all. Such scenarios are considered
+"custom" scenarios and can be created and easily manipulated by the user.
+Examples include:
+
+- Different CTI options
+- Different "cold attitude" options
+- What if's, like what if we didn't manage to start the maneuver to 135 pitch in a NSM recovery?
+
+One special scenario is the "flight" scenario, discussed below.
+
+Flight scenario (no network)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+In the case of production applications that are running on the HEAD or GRETA
+networks or where it is not possible or desirable to access the network to
+update the local loads, the special ``"flight"`` scenario can be used.
+
+The impact of selecting ``scenario="flight"`` in a commands query is that it
+disables access to network resources (Google sheets and OCCweb). This means that
+results will depend strictly on the production commands archive files
+``${SKA}/data/kadi/cmds2.h5`` and ``${SKA}/data/kadi/cmds2.pkl``. On the HEAD
+network these files are brought up to date each 10 minutes by a cron jobs, so
+using ``"flight"`` in this case is a reliable way to eliminate dependence on the
+kadi external web resources.
+
+Using the ``"flight"`` scenario is also recommended for use on GRETA
+workstations since they cannot access the Chandra Command Events Google sheet.
+
+Custom scenario example
+^^^^^^^^^^^^^^^^^^^^^^^
+
+This example shows the steps to programmatically add an ACIS CTI in the midst
+of the 2021:296 NSM recovery::
+
+    # For Ska3 2022.2 this is necessary
+    >>> import sys
+    >>> sys.path.insert(0, '/Users/aldcroft/git/parse_cm')
```

Comment:
I don't see how this PR is going to get promoted with the parse_cm one too so...

Reply:
Fixed.